types/blob_txn.go

```diff
@@ -122,6 +122,7 @@ type BlobTx struct {
 	Creation               bool // true if To field is nil, indicating contract creation
 	Value                  uint256.Int
 	DataLen                int // length of the Data in bytes
+	DataNonZeroLen         int
```

Comment:
Curious why we need this?

Reply:
Non zero byres are priced differently so the txpool needs it to compute gas fees.